frontend/src/pages/onboarding.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from 'react';
+import { Button, ButtonGroup, Flex } from '@chakra-ui/react';
+import FormContainer from '../components/FormContainer';
+import MenteeForm from '../components/MenteeForm';
+import MentorForm from '../components/MentorForm';
+
+interface Persona {
+  role: string;
+}
+
+const Onboarding = () => {
+  let [persona, setPersona] = useState<Persona>({
+    role: '',
+  });
+
+  const menteeFields = [
+    {
+      inputType: 'textArea',
+      label: 'Tell us a little about yourself!',
+      name: 'bio',
+    },
+    {
+      inputType: 'textArea',
+      label: 'What are you looking to learn?',
+      name: 'learningGoals',
+    },
+    {
+      inputType: 'input',
+      label: 'How many years of experience do you have as a software engineer?',
+      name: 'experience',
+    },
+    {
+      inputType: 'input',
+      label: 'What is your current job title?',
+      name: 'jobTitle',
+    },
+    {
+      inputType: 'textArea',
+      label: 'What are your interests?',
+      name: 'interests',
+    },
+  ];
+
+  const mentorFields = [
+    {
+      inputType: 'textArea',
+      label: 'Tell us a little about yourself!',
+      name: 'bio',
+    },
+    {
+      inputType: 'textArea',
+      label: 'What are your areas of expertise?',
+      name: 'expertise',
+    },
+    {
+      inputType: 'input',
+      label: 'How many years of experience do you have as a software engineer?',
+      name: 'experience',
+    },
+    {
+      inputType: 'input',
+      label: 'What is your current job title?',
+      name: 'jobTitle',
+    },
+    {
+      inputType: 'textArea',
+      label: 'What are your interests?',
+      name: 'interests',
+    },
+    {
+      inputType: 'textArea',
+      label: 'What is your hourly rate? Leave as 0 for free mentoring.',
+      name: 'hourlyRate',
+    },
+  ];
+
```

Comment:
Export all of the above into a `constants.ts` file (put in src root) and change casing to UPPER_CASE. Can also create one variable for shared attributes and then extend the required ones for mentee/mentor

Reply:
The following has been completed:

- Created constants.tsx
- Created one variable called QUESTIONNAIRE_FIELDS for the shared attributes and extended this variable for each mentee and mentor variable, spreading the QUESTIONNAIRE_FIELDS into each.
- Exported the mentor and mentee variables and imported those into onbaording.tsx and referenced those variables when passing props to the respective child components.
- Deleted the mentee and mentor variables on onboarding.tsx